config/crd/patches/webhook_in_addons.yaml

```diff
@@ -1,6 +1,6 @@
 # The following patch enables conversion webhook for CRD
 # CRD conversion requires k8s 1.13 or later.
```

Comment:
```suggestion
# CRD conversion requires k8s 1.16 or later.
```

Reply:
we actually goes to 1.17. but, the comments are not blocker take it.